indico/modules/events/surveys/controllers/management/questionnaire.py

```diff
@@ -188,14 +276,25 @@ def _process(self):
         except KeyError:
             raise NotFound
 
-        form = field_cls.config_form()
+        try:
+            clone_id = int(request.args.get('clone'))
+        except ValueError:
+            clone_id = None
+
```

Comment:
Could use an `else` block here instead of checking whether the id is not None below. Also, `int(None)` raises a `TypeError` so this will fail when the clone argument is not preset.